drag_and_drop_v2/public/css/drag_and_drop_edit.css

```diff
@@ -271,7 +271,8 @@
 
 .xblock--drag-and-drop--editor .tab .field-error {
     outline: none;
-    box-shadow: 0 0 10px darkred;
+    box-shadow: 0 0 10px 0 darkred;
+    -webkit-appearance: none;
```

Comment:
Nit: Please add a comment explaining why this style is here... Otherwise someone might remove it in a future refactor.